css/backoffice/blocks-integrations/_summary-card-within-tippy.scss

```
@@ -0,0 +1,15 @@
+/*
+ * @copyright   Copyright (C) 2010-2023 Combodo SARL
+ * @license     http://opensource.org/licenses/AGPL-3.0
+ */
+
+$ibo-vendors-tippy--summary--background-color: $ibo-color-transparent !default;
+$ibo-vendors-tippy--summary--color: $ibo-body-text-color !default;
+
+.tippy-box[data-theme~='summary'] {
+  background-color: $ibo-vendors-tippy--summary--background-color;
+  color: $ibo-vendors-tippy--summary--color;
```

```suggestion
$ibo-vendors-tippy--object-summary--background-color: $ibo-color-transparent !default;
$ibo-vendors-tippy--object-summary--color: $ibo-body-text-color !default;

.tippy-box[data-theme~='object-summary'] {
  background-color: $ibo-vendors-tippy--object-summary--background-color;
  color: $ibo-vendors-tippy--object-summary--color;
```
And rename file "_object-summary-within-tippy.scss".

This way it is more compliant with the "object-details" and "object-summary" blocks.

---

Done in [13320b8](https://github.com/Combodo/iTop/pull/476/commits/13320b87bbb5c3492b82ba76ba92d14f2dfac4be)